org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/extensions/dtd/contentmodel/CMDTDAttributeDeclaration.java

```diff
@@ -45,7 +55,14 @@ public String getDefaultValue() {
 
 	@Override
 	public String getDocumentation() {
-		return null;
+		if (documentation != null) {
+			return documentation;
+		}
+		Map<String, String> commentsMap = document.getCommentsMap();

```

Comment:
I think CMDTDElementDeclaration should provide a new method `CMDTDElementDeclaration#getDocumentation(String attrName)`.

Replace with 

`this.documentation = elementDecl.getDocumentation(getName())`